### DISCUSSION TITLE
Website crashed after paying for a premium license

### DISCUSSION THREAD

**Discussion by Waleed-Elmughrabi** [aclap-dev/video-downloadhelper#1394] [License & Purchase requests]
### Your question

Hi there, just paid for a premium license. After making the purchase, I can confirm that money has been paid out of bank account, but then website crashed before getting to the confirmation page. I didn't receive any emails as well, checked inbox and spam folder.

Can you please reach out.

Thanks

### Addon & CoApp version and details

version: 9.1.1.0
target: google
channel: stable
lang: en-GB
coapp: {"found":true,"new_version":false,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19"}
license: {"unset":true}
platform: arm64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36
{
    "used_history_button": true,
    "download_directory": "/Users/w/dwhelper",
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 7,
    "successfull_dl": 5
}

**Comment by leandrorovagnoli** [comment]
Hi there!
I just get caught in the same situation. Just paid, website crashed with a "timeout" message and my money went throught from my bank account. Can anyone help me as well?

**Comment by jcfrog** [comment]
Hi, sorry for these incidents, please send your email addresses at licenses@downloadhelper.net, we will refund the transactions that failed.

**Comment by Glaverbel** [comment]
Hi, 
I just paid through Google Pay, the website crashed with a "timeout" message, but the credit card company sent me a note that the payment is successful. Can anyone help on this?
